package.json

```diff
@@ -18,8 +18,16 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "flow": "flow; test $? -eq 0 -o $? -eq 2"
+    "flow": "flow; test $? -eq 0 -o $? -eq 2",
+    "lint": "eslint ./ --cache --ignore-pattern .gitignore",
+    "prettify": "prettier --write --tab-width 4 --trailing-comma es5 --single-quote \"src/**/*.js\"",
```

Comment:
Just tried to follow the original spacing to lessen prettier's footprint.
Well, since prettier has already made its input into the code, will switch spacing to 2.